CONTRIBUTING.rst

```diff
@@ -20,8 +20,8 @@ Code Style
 ----------
 
 Although it is not strictly enforced yet, we strongly suggest to follow the
-`pep8 coding standards <https://www.python.org/dev/peps/pep-0008/>`_.
-Two popular modules used to check pep8 are
+`PEP8 coding standards <https://www.python.org/dev/peps/pep-0008/>`_.
+Two popular modules used to check pep8 compliance are
```

Comment:
Can we include [black](https://pypi.org/project/black/) here, too? I've been using black with pysteps for some time now and I'm pretty happy with it.

it can be installed using
`pip install black`

or alternatively

`conda install -c conda-forge black`

Reply:
although we should probably specify that black, unlike pylint and pycodestyle, is an autoformatter.

Reply:
Good idea, I added an "Auto-formatters" section, where black is recommended. 6f9b0405c4b4fce428bc92274a11381583108a4a

Reply:
An interesting fact: black  is used in the [pylint](https://github.com/PyCQA/pylint) project, that checks PEP8 compliance, to check and autoformat for PEP8 compliance in its own code :satisfied:

Another good reason to use black.